package.json

```diff
@@ -29,8 +29,9 @@
     "build:client": "cd ./client && npm run build",
     "build:curriculum": "cd ./curriculum && npm run build",
     "build:server": "cd ./api-server && npm run build",
-    "clean": "npm-run-all clean:client clean:server clean:packages clean:root-deps",
+    "clean": "npm-run-all clean:build-files clean:client clean:server clean:packages clean:root-deps",
     "clean-and-develop": "npm run clean && npm ci && npm run develop",
+    "clean:build-files": "shx rm ./tools/scripts/build/tsconfig.tsbuildinfo",
```

Comment:
@ojeytonwilliams Is this what you had in mind? Should I instead write a script for this, and include all the build files in it?

Reply:
This looks good, I don't think we need anything more sophisticated.